cmd/kubeops/main.go

```diff
@@ -228,5 +229,8 @@ func parseClusterConfig(configPath, caFilesPrefix string) (kube.ClustersConfig,
 		}
 		configs.Clusters[c.Name] = c
 	}
+	if configs.KubeappsClusterName == "" {
+		return kube.ClustersConfig{}, deferFn, fmt.Errorf("unable to determine which cluster Kubeapps is installed in")
+	}
```

Comment:
Similar here: there is a valid use-case where we don't have a name in the cluster configs for the cluster on which Kubeapps is installed.